.cargo/config.toml

```diff
@@ -1,23 +1,23 @@
 [target.xtensa-esp32-espidf]
 linker = "ldproxy"
 runner = "espflash --monitor"
-rustflags = ["-C", "default-linker-libraries", "-Clink-args=-Wl,-Map=./esp-build.map"]
+rustflags = ["-C", "default-linker-libraries", "-Clink-args=-Wl,-Map=./esp-build.map", "--cfg", "espidf_time32"]
```

Comment:
Required due to `libc` crate upgrade and https://github.com/rust-lang/libc/pull/3993.